webapp/webapp/demo/views.py

```diff
@@ -15,8 +17,11 @@
 
 vocab_path = os.getenv('VOCAB_PATH', '/tmp/vocab.dat')
 cdb_path = os.getenv('CDB_PATH', '/tmp/cdb.dat')
-auth_callback_service = 'https://medcat.rosalind.kcl.ac.uk/auth_callback'
-validation_base_url = 'https://uts-ws.nlm.nih.gov/rest/isValidServiceValidate'
+umlssm_mp_path = os.getenv('UMLSSM_MP_PATH', '/medcat_data/umls_sm_wstatus_2021_oct.zip.zip')
```

Comment:
Does this mean if I want to add new models we have to do it here?

Reply:
Would be good to have a small django admin where you can add/remove models

Reply:
Good idea. I thought this epic did not cover model management or anything alike. Will think about that.